tests/runtime/call

```diff
@@ -412,3 +412,15 @@ EXPECT OK
 REQUIRES_FEATURE skboutput
 TIMEOUT 5
 MIN_KERNEL 5.5
+
+NAME debugf
+REQUIRES echo > /sys/kernel/debug/tracing/trace
```

Comment:
`REQUIRES` should be used to check for existence of resources, use `BEFORE` to prepare a test environment.